pjsip/include/pjsua2/endpoint.hpp

```diff
@@ -1162,11 +1162,24 @@ struct EpConfig : public PersistentObject
 /* This represents posted job */
 struct PendingJob
 {
+    PendingJob():autoDelete(true){};
+
+    PendingJob(bool autoDel):autoDelete(autoDel) {};
+
     /** Perform the job */
     virtual void execute(bool is_pending) = 0;
 
+    bool getAutoDelete() { return autoDelete; };
```

Comment:
IMO if this method is only useful for `Endpoint`, perhaps use class `friend` feature is cleaner?